recipe/meta.yaml

```diff
@@ -39,11 +42,11 @@ test:
   commands:
     # Skip win: false positive jupyter-server 1.13.5 has requirement pywinpty<2 but you have pywinpty 2.0.2.
     - python -m pip check  # [not win]
-#  downstreams:
+  downstreams:
```

Comment:
please remove this downstream feature ...

Reply:
Yep, it's only for testing. I've commented it.